app/search/classification/base.py

```diff
@@ -0,0 +1,27 @@
+from urllib.parse import urlparse
+
+
+class BaseClassifier(object):
+    """
+    TODO: Classifiers are...
```

Comment:
I'll get there before merging. Wanted to get this up for you tonight, in case you had some free time to take a look.